app/__init__.py

```diff
@@ -9,22 +8,24 @@
 
 # # local  imports
 from instance.config import app_config
-# from .database_config import create_tables
-# create_tables()
+from .database_config import create_tables, create_test_tables
```

Comment:
we talked about this last time that your avoid relative imports

Reply:
I'm doing that with immediate effect. My apologies I didn't remove all.